app/views/users/_follow.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_for current_relationship, remote: true do |f| %>
```

Comment:
remote: true dùng để làm gì

Reply:
dạ
khi thêm remote : true thì form sẽ submit hoặc gọi hành động mà đã được định nghĩa trong form

Reply:
`form sẽ submit ` 
ủa chị ko có remote true thì bấm submit form mà, cái này phải submit form mô gái, tìm hiểu lại

Reply:
Gởi dữ liệu vào form mà k load loại trang thông qua ajax 

Thực hiện gọi ajax in rails bằng remote: true